### PR TITLE
Filter trainer UI

### PR DESCRIPTION
**Pull request by Ponrawoot**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,10 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
     "@headlessui/react": "^1.7.10",
     "@heroicons/react": "^2.0.14",
+    "@mui/icons-material": "^5.11.0",
```
Comment (noppakorn) [inline package.json:15]: why?

File: src/pages/user/booking/index.tsx [hunk 2]
```diff
@@ -12,6 +12,7 @@ import {
 import Link from "next/link";
 import { useRouter } from "next/router";
 import { NavBar } from "@/components/navbar";
+import Rating from "@mui/material/Rating";
```
Comment (noppakorn) [inline src/pages/user/booking/index.tsx:15]: why?

File: src/pages/_app.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ export default function App({ Component, pageProps }: AppProps) {
 
   // Check if that path is protected, then check if the user is logged in
   const authCheck = (url: string) => {
-    const publicPaths = ["/", "/account/signup"];
+    const publicPaths = ["/", "/account/signup","/user/booking"];
```
Comment (noppakorn) [inline src/pages/_app.tsx:29]: please do not forget about this

**Review by noppakorn — CHANGES_REQUESTED** [review]

**Review by nnanthicha — CHANGES_REQUESTED** [review]
merge main into your branch and update code

**Review by noppakorn — CHANGES_REQUESTED** [review]
Same as nan, merge the main branch into your code.

**Review by noppakorn — APPROVED** [review]

**Pull request merged**